### DISCUSSION TITLE
how clear hover

### DISCUSSION THREAD

**Discussion by devopg** [tailwindlabs/tailwindcss#10291] [Help] [answered]
Hello. I have UI framework which generate many styles on hover by tailwind but i can rewrite styles.
Could tailwind claer all styles on the hover for my component?\
like this 
`<myComponent class="hover:none"></myComponent>`

**Comment by wongjn** [comment, marked as answer]
You could use the [`all` shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/all) with the value of [`revert`](https://developer.mozilla.org/en-US/docs/Web/CSS/revert), [`initial`](https://developer.mozilla.org/en-US/docs/Web/CSS/initial) or [`unset`](https://developer.mozilla.org/en-US/docs/Web/CSS/unset) (depending on which behavior you desire) via an [arbitrary property class](https://tailwindcss.com/docs/adding-custom-styles#arbitrary-properties) like `hover:[all:unset]`

**Reply by devopg** [reply]
oh i dont know that 'all' there are.
